README.md

```diff
@@ -7,10 +7,137 @@ new [cisagov](https://github.com/cisagov) Go project started.
+with all available dependencies for this project you can run this
+command in the project root:
+
+```console
+nix develop
+```
+
+If you make changes to the project files you will need to update `vendorSha256`
+in `flake.nix`  You can do this by running `nix build` and you should be presented
+with an error similar to:
+
+```console
+error: hash mismatch in fixed-output derivation '/nix/store/11a71sp1wynjgxinx6yb0yhli4q659zi-skeleton-golang-package-20230424-go-modules.drv':
+         specified: sha256-+aBatR0sHWkykrQr8AMD0P5xoc4VEXtA+egp6PxsKzY=
+            got:    sha256-gqfvjULp2VApWQl7yFVj45meYpcS4XefUtEUy+TtAH4=
+error: 1 dependencies of derivation '/nix/store/nm6xhgfawf67sy89rl9azdzvnjc5r7cr-skeleton-golang-package-20230424.drv' failed to build
+```
+
+You would then copy the `got` hash and use it to replace the previous
+ `vendorSha256` value within `flake.nix`.
+
+If `flake.nix` changes or is modified you will also need to update the
+`flake.lock` file. You can do this by running: `nix flake update`. It is
+also recommended to then run `nix build` after this operation to ensure
+that the build is successful.
+
+## SBOM ##
+
+A Software Bill of Materials (SBOM) is like a recipe card for software,
+detailing all the ingredients (components, libraries, dependencies)
+used to cook up a digital product. It's a crucial tool for understanding
+what goes into software, helping with things like spotting vulnerabilities,
+ensuring license compliance, and keeping an eye on supply chain security.
+However, SBOM isn't a magic fix for security issues itself—it's more of a
+roadmap that helps teams navigate potential risks. It's not a substitute
+for good coding practices or regular updates, but a helpful companion in
+the journey towards stronger software security.
+
+The SBOM found in this project was generated using [cyclonedx-gomod](https://github.com/CycloneDX/cyclonedx-gomod).
+The exact commands used can be found below.
+
+```console
+cyclonedx-gomod mod -std -json -output bom.json  ./
+```
+
+Generates an SBOM in JSON format, includes the aggregate of modules
+required by all packages in the target module. This optionally
+includes modules required by tests and test packages. **Build
+constraints are not evaluated**, allowing for a "whole picture"
+view on the target module's dependencies.
+
+```console
+cyclonedx-gomod app -std -json -output sbom.json  ./
+```
+
+Generates an SBOM in JSON format for only the
+modules the target application depends on. Modules required by tests
+or packages that are not imported by the application are not included.
+**Build constraints are evaluated**, which enables a very detailed view of
+what's really compiled into an application's binary.
+
+This project does not inlude executables but if that changes in the future
+or if the application is modified to include additional dependencies or updates
+the SBOM should be updated to reflect that.
```

Comment:
Since the SBOM files were removed in https://github.com/cisagov/skeleton-golang-package/pull/2/commits/8065662c1f615c947757bd68005c6cb6b917fb48, this "SBOM" section in the README should also be removed.